schema/appState.json

```diff
@@ -0,0 +1,507 @@
+                    "type": ["null", "string"]
+                },
+                "error": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/WodinError" }
+                    ]
+                }
+            }
+        },
+
+        "FitData": {
+            "type": "object",
+            "additionalProperties": {
+                "$ref": "#/definitions/NumberArray"
+            }
+        },
+
+        "ModelFitState": {
+            "type": "object",
+            "properties": {
+                "fitting": {
+                    "type": "boolean"
+                },
+                "fitUpdateRequired": {
+                    "type": "boolean"
+                },
+                "iterations": {
+                    "type": ["null", "number"]
+                },
+                "converged": {
+                    "type": ["null", "boolean"]
+                },
+                "sumOfSquares": {
+                    "type": ["null", "number"]
+                },
+                "paramsToVary": {
+                    "$ref": "#/definitions/StringArray"
+                },
+                "result": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/OdinFitResult" }
+                    ]
+                },
+
+            }
+        },
+
+        "OdinFitResult": {
+            "type": "object",
+            "properties": {
+                "inputs": {
+                    "$ref": "#/definitions/OdinFitInputs"
+                },
+                "hasResult": {
+                    "type": "boolean"
+                },
+                "error": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/WodinError" }
+                    ]
+                }
+            }
+        },
+
+        "OdinFitInputs": {
+            "type": "object",
+            "properties": {
+                "endTime": {
+                    "type": "number"
+                },
+                "link": {
+                    "$ref": "#/definitions/FitDataLink"
+                },
+                "data": {
+                    "$ref": "#/definitions/FitData"
+                },
+                "parameterValues": {
+                    "$ref": "#/definitions/OdinParameterValues"
+                }
+            }
+        },
+
+        "FitDataLink": {
+            "type": "object",
+            "properties": {
+                "time": {
+                    "type": "string"
+                },
+                "data": {
+                    "type": "string"
+                },
+                "model": {
+                    "type": "string"
+                }
+            },
+            "required": ["time", "data", "model"],
+            "additionalProperties": false
+        }
+    },
+
+    "properties": {
```

Comment:
Don't we need `modelFit` as a property here as well?

Reply:
Added in 6280409